### PR TITLE
JEN-1022 add separate jobs for ps fb

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

**Review by EvgeniyPatlan — APPROVED** [review]
LGTM

File: jenkins/param-fb.yml [hunk 1]
```diff
@@ -0,0 +1,121 @@
+- job:
+    name: percona-server-8.0-fb-param
```
Comment (inikep) [inline jenkins/param-fb.yml]: This job has nothing in common with Percona Server. It should be `fb-mysql-server-8.0` or more generic `mysql-server-8.0` (for testing both MySQL and FB-MySQL).

Comment (hors) [reply jenkins/param-fb.yml]: ok, I see your point. I have fixed it.

File: jenkins/pipeline-fb.groovy [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+if (params.ANALYZER_OPTS.contains('-DWITH_VALGRIND=ON'))
+    { pipeline_timeout = 144 }
+
+if (
+    ((params.ANALYZER_OPTS.contains('-DWITH_ASAN=ON')) &&
+    (params.ANALYZER_OPTS.contains('-DWITH_ASAN_SCOPE=ON')) &&
+    (params.ANALYZER_OPTS.contains('-DWITH_UBSAN=ON'))) ||
+    ((params.MTR_ARGS.contains('--big-test')) || (params.MTR_ARGS.contains('--only-big-test')))
+    ) {
+        LABEL = 'docker-32gb'
```
Comment (percona-ysorokin) [inline jenkins/pipeline-fb.groovy]: This `LABEL = 'docker-32gb'` is probably not needed here as we always use `docker-32gb`
```suggestion
```

Comment (hors) [reply jenkins/pipeline-fb.groovy]: yes, you are right we can remove it

**Review by percona-ysorokin — APPROVED** [review]
LGTM

**Pull request merged**